subprojects/core/src/main/java/org/gradle/api/internal/changedetection/state/DefaultExecutionHistoryCacheAccess.java

```diff
@@ -27,9 +27,9 @@
 public class DefaultExecutionHistoryCacheAccess implements ExecutionHistoryCacheAccess, Closeable {
     private final PersistentCache cache;
 
-    public DefaultExecutionHistoryCacheAccess(ScopedCache cacheRepository) {
-        this.cache = cacheRepository
-            .cache("executionHistory")
+    public DefaultExecutionHistoryCacheAccess(ScopedCacheBuilderFactory cacheBuilderFactor) {
```

Comment:
I think this should be `cacheBuilderFactory`

Reply:
I'm seriously impressed you noticed this, looking at so many similar small changes!